### PR TITLE
Added more documentation

### PR DESCRIPTION
**Pull request by kerim1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,293 @@
...
+## How the service is built up
+
+Resussun is a service written in Java using the [Dropwizard](#https://www.dropwizard.io) framework. 
+It makes use of [ElasticSearch](#https://www.elastic.co/elasticsearch) as the search engine 
+and [Redis](#https://redis.io) for the storage of metadata and to provide caching.
+
+![](./docs/components.png)
+
+Through the admin interface an administrator can create reconciliation services for any number of configured datasets 
+from various Timbuctoo instances. Resussun will use the [GraphQL](#https://graphql.org) interface
```
Comment (MMaas3) [inline README.md]: We might want to add a link to Timbuctoo.

**Comment by MMaas3** [comment]
Another thing I miss is an example on how to create an index for a Reconciliation API.

File: README.md (hunk 1 above)
Comment (kerim1) [reply README.md]: Link added.

**Comment by kerim1** [comment]
Added an extra paragraph to the example on how to create the index.

File: README.md (hunk 1 above)
Comment (MMaas3) [reply README.md]: Just now I see that the external links do not work. The `#` should be removed.

**Pull request merged**